importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -4842,7 +4871,7 @@ Note* MusicXMLParserPass2::note(const QString& partId,
             else if (_e.name() == "stem")
                   stem(stemDir, noStem);
             else if (_e.name() == "type") {
-                  small = _e.attributes().value("size") == "cue";
+                  small = _e.attributes().value("size") == "cue" | _e.attributes().value("size") == "grace-cue";
```

Comment:
This causes a compiler warning due to mussing parentheses. Alsi IU guess it shpould be `||`` rather than `|`?

Reply:
#8554 (for posterity)